### ISSUE TITLE
add name_description for the disk

### ISSUE THREAD

**Issue by moisei** [terra-farm/terraform-provider-xenorchestra#28]
Would be nice to have an option to specify disk description in the terraform script.
Something like:

```
resource "xenorchestra_vm" "vm1" {
...
 disk {
    sr_id      = "111111111-22222-333-444-555555555"
    name_label = var.vm_name
    name_description = "MY DISK DESCRIPTION"
    size = 21474836480
  }
}

```
Thank you.

**Comment by ddelnano** [comment]
With the latest improvements made in #89, this is more approachable to implement.

**Comment by ddelnano** [comment]
@moisei this has been implemented in v0.11.0 of the provider.
